taipy/gui/gui.py

```diff
@@ -2245,14 +2256,30 @@ def _notify(
         message: str = "",
         system_notification: t.Optional[bool] = None,
         duration: t.Optional[int] = None,
+        notification_id: str = "",
     ):
         self.__send_ws_alert(
             notification_type,
             message,
             self._get_config("system_notification", False) if system_notification is None else system_notification,
             self._get_config("notification_duration", 3000) if duration is None else duration,
+            notification_id,
         )
 
+    def _close_notification(
+        self,
+        notification_id: str,
+    ):
+        if notification_id:
+            self.__send_ws_alert(
```

Comment:
why not call `_notify` ?

Reply:
The _notify method creates and sends a notification to the user interface, configuring its appearance The _close_notification method focuses solely on closing an existing notification i can use _notify also by changing some logic introducing new close bool option should i do that?

Reply:
ok